doc/classes/NavigationAgent2D.xml

```diff
@@ -229,17 +230,17 @@
 		</signal>
 		<signal name="navigation_finished">
 			<description>
-				Notifies when the final position is reached.
+				Emitted once per loaded path when the agent internal navigation path index reaches the last index of the loaded path array. The agent internal navigation path index can be received with [method get_current_navigation_path_index].
 			</description>
 		</signal>
 		<signal name="path_changed">
 			<description>
-				Notifies when the navigation path changes.
+				Emitted when the agent had to update the loaded path. E.g. because the path was previously empty or due to a navigation map change or being pushed further away from the current path segment than the [member path_max_distance] allows.
```

Comment:
if bullet point lists are available consider using it indeed of writing X or Y or Z